src/main/resources/org/jenkinsci/plugins/oic/OicSecurityRealm/help-clientSecret.html

```diff
@@ -1,4 +1,5 @@
 <div>
-	The Client secret used by jenkins to authenticate itself to the openid connect provider.
-	Special value "none" can be used to disable client secret (only with client_secret_post authentication).
+	The Subject Idendifier Type to use as part of the flow with the openid connect provider.
+	This must be supported by the IdP.
+	Valid values are <code>public</code> and <code>pairwise</code>. 
```

Comment:
I think this PR is out of date with the target branch (or incorrect merge has been performed?)